p5js/ImageClassification/ImageClassification_Teachable-Machine/sketch.js

```diff
@@ -5,47 +5,64 @@
 
 /* ===
 ml5 Example
-Webcam Image Classification using a pre-trianed customized model and p5.js
+Webcam Image Classification using a pre-trained customized model and p5.js
 This example uses p5 preload function to create the classifier
 === */
 
-// const checkpoint = 'model/model.json';
-const checkpoint = 'https://storage.googleapis.com/teachable-machine-pubilshed-models/fa542ec0-b94b-4aa9-add4-c256963b0720/model.json';
+// Classifier Variable
 let classifier;
+// Model URL
+let imageModel = 'https://storage.googleapis.com/teachable-machine-pubilshed-models/fa542ec0-b94b-4aa9-add4-c256963b0720/model.json';
```

Comment:
Maybe it makes sense to call it `imageModelURL` ? Just for clarity?

Reply:
Great point! changing.